pkgs/tools/typesetting/biber/default.nix

```diff
@@ -10,6 +10,20 @@ perlPackages.buildPerlModule {
 
   src = "${biberSource}/source/bibtex/biber/biblatex-biber.tar.gz";
 
+  patches = [
```

Comment:
maybe
```diff
- patches = [
+ patches = lib.optionals (lib.versionAtLeast perl.version "5.30") [`
```
?
just in case someone will use it with perl 5.28 (by cherrypicking it to nixos-19.03 for example)

Reply:
@volth good idea - I updated the PR. feel free to merge if you agree.